modules/common/tests/src/Functional/Api1TestBase.php

```diff
@@ -58,10 +62,11 @@ protected function assertJsonIsValid($schema, $json) {
   abstract public function getEndpoint(): string;
 
   protected function post($data, $httpErrors = TRUE) {
-    return $this->http->post($this->endpoint, [
+    return $this->httpClient->post($this->endpoint, [
       RequestOptions::JSON => $data,
       RequestOptions::AUTH => $this->auth,
       RequestOptions::HTTP_ERRORS => $httpErrors,
+      RequestOptions::TIMEOUT => 10000,
```

Comment:
@dafeder Just for my understanding, did you run into an issue with timeout which resulted in adding this line?

Reply:
@jastraat oh huh, I forgot I left that in there. Not in a normal test run, but when using the debugger the request would timeout before I had a chance to inspect what was going on at different breakpoints, which this fixed. I should probably clean that up though.